### DISCUSSION TITLE
Using ffmpeg to download embedded video created by VdoCipher

### DISCUSSION THREAD

**Discussion by quickmaths1** [aclap-dev/video-downloadhelper#499] [Help and Q&A]
### Your question

I have videos that I would like to download from a paid website that uses VdoCipher Player to post their videos to their website. Is it possible to download the video using ffmpeg? I don't think video download helper extension will work in this case. The image shows the video when I right click and click "About VdoCipher Player." The video is about 8 hrs long and should probably come out to like 1gb according to vdh extension when I tried using it to download the video. 

Thanks.

![share_6221225695873483584](https://github.com/aclap-dev/video-downloadhelper/assets/170026625/e079ea0e-6344-47b4-953f-1bbd8ed6e0a1)

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: no.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Since you're talking about a web site that requires a subscription, that is going to hinder people here from helping you.  Still, I have some observations.  First off, you are not using the latest version of VDH.  Go back to the main discussion page of this forum.  Look at the top of the page.  There is an announcement of VDH version 9.x.  You need to get the latest version of that.  It's a beta.  Never mind about that.  It is the latest version & there is always a chance it will work better than the outdated version you are running.

I notice you are not using the CoApp.  It might help.  Maybe.  Maybe not.  But you can't know unless you try it.  The CoApp is pretty much required for most content around the web these days.  I don't understand the reluctance of people to use it.  Try it.  See if it makes a difference.

Even though your content is behind a paywall, there is a way that lets non-subscribers access the content.  SOMETIMES.  I don't want to oversell this.  Sometimes it works, sometimes it doesn't.  Again, you can't know unless you try it.  I explain the technique at length over here: #485.

Using ffmpeg is a bit off-topic for this forum.  I use it regularly but any problems I have with it, I keep to myself, work on myself, correct myself, fail to correct myself, & I don't post about it here.  Yes, ffmpeg is an integral part of the CoApp.  If you follow my advice & try the CoApp, you will actually be using ffmpeg.  Maybe that will answer your question about whether ffmpeg might succeed on your content.  But I'm not going to help you debug any trouble you have with ffmpeg.  You are strictly on your own with that.  The closest I will come to helping you with ffmepg is this: #497.

Happy reading!
